src/language/norwegian.jsx

```diff
@@ -4,7 +4,9 @@ export default {
     APP_TITLE: 'NAV ikon-base',
     DOWNLOAD_ICON: "Last ned ikon",
     FILLED_ICON: 'Filled',
+    ICON_IN_LIST: 'icon-in-list',
```

Comment:
Dette er vel ikke tekst som skal vises, men heller klassenavn? Ville isåfall lagt de inn der de brukes for å ikke forvirre de som evt. skal endre på tekster i ettertid